aerogear/mobile-developer-console
# Added code for version of MDC, fixed bug with version for about modal, & add RH Mobile SVG.

## Motivation
Added MDC version to about modal.

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Install on RHPDS system
2. Verify that version shows up in the about modal.

## Checklist:


## Progress

- [ ] TODO: Still need to verify on RHPDS.

## Additional Notes

Do not merge until I have verified the version shows up correctly.
